lib/key.js

```diff
@@ -0,0 +1,51 @@
+const { readFile } = require("fs");
+const crypto = require("crypto");
```

Comment:
May use bluebird/promise promisify

Reply:
Sure but I think direct usage is still fine